packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +102,38 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * TransactionReceipt representation
+ * @property effectiveGasPrice - Effective gas price the transaction was charged at
+ * @property gasUsed - Gas used in the transaction
+ * @property l1Fee - Total used gas in hex
+ * @property status - The status of the transaction
+ */
+export interface TransactionReceipt {
+  blockHash?: string;
+  blockNumber?: number;
+  effectiveGasPrice?: string;
+  gasUsed?: string;
+  l1Fee?: string;
+  logs?: Log[];
+  status?: string;
+  transactionIndex?: number;
+}
+
+/**
+ * @type Log
+ *
+ * Log representation
+ * @property address - Address of the contract that generated log
```

Comment:
Worth moving inside the interface?

Reply:
Fixed in https://github.com/MetaMask/core/pull/1592/commits/c84227169c0c8b168a3171b3048b4939a1d07411